notes/markov.md

```diff
@@ -40,7 +91,7 @@ The adjacency matrix, <span>\\({\bf A}\\)</span>, for directed graphs is defined
 
 <div>\[ a_{ij} = \begin{cases} 1 \quad \mathrm{if} \ \mathrm{node}_i \leftarrow \mathrm{node}_j \\ 0 \quad \mathrm{otherwise} \end{cases}, \]</div>
 
-where <span>\\(a_{ij}\\)</span> is the <span>\\((i,j)\\)</span> element of <span>\\({\bf A}\\)</span>. The adjacency matrix which describes the example graph above is:
+where <span>\\(a_{ij}\\)</span> is the <span>\\((i,j)\\)</span> element of <span>\\({\bf A}\\)</span>. This matrix is typically asymmetric, so it is important to adhere to the definition. The adjacency matrix which describes the example graph above is:
```

Comment:
may want to note that which of the to/from connections are the *columns* and which are the *rows* to be extra explicit

Reply:
Also do we want to note/care about self-loops/multiple copies of an edge here?

Reply:
also note that what the rows/cols represent in a markov matrix isn't always standardized (some use the reverse of 357 notation)